src/python/pants/option/options_bootstrapper.py

```diff
@@ -167,6 +169,9 @@ def filecontent_for(path: str) -> FileContent:
             args = alias.expand_args(tuple(args))
             bargs = cls._get_bootstrap_args(args)
 
+            # @TODO: ...
+            os.environ["PANTS_BIN_NAME"] = bootstrap_option_values.pants_bin_name
```

Comment:
This isn't a great place for this, but it happens very early on in the process. There's only 2 call sites, I might just move it.

Reply:
This actually seems fine. It's where our options setup takes place, it looks like.